lib/connect.js

```diff
@@ -593,4 +598,32 @@ function normalizeVideoCodecSettings(nameOrSettings) {
   }
 }
 
+function validateBandwidthProfile(bandwidthProfile) {
+  if (typeof bandwidthProfile !== 'object' || Array.isArray(bandwidthProfile)) {
```

Comment:
```suggestion
  if (bandwidthProfile === null || typeof bandwidthProfile !== 'object' || Array.isArray(bandwidthProfile) ) {
```

also since we are using such check in lots of places, lets add `isObject` function in utils and use it here.